src/ts/common/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { LottoConfig, RewardByMatchCnt } from "./constants";
+import { Lotto, WinningLotto } from "./interfaces";
+
+export function getRandomInt(min: number, max: number): number {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+export function shuffle(arr: any[]) {
+  for (let i = 0; i < arr.length; i++) {
+    const j = getRandomInt(0, arr.length - 1);
+    [arr[i], arr[j]] = [arr[j], arr[i]];
+  }
```

Comment:
```suggestion
arr.map((_, i) => {
  const j = getRandomInt(0, arr.length - 1);
  [arr[i], arr[j]] = [arr[j], arr[i]];
})
```
일반 for 반복문의 경우 변수 선언, 부등호 비교, 증감식 등 사람이 실수할 수 있는 요소가 많습니다. 그래서 가능하면 Array내장객체의 메서드를 이용해서 반복하는 방법도 있습니다. 😄